packages/ui/src/memberships/components/MemberRoles.tsx

```diff
@@ -101,7 +102,8 @@ export interface MemberRolePopupContentProps {
 }
 
 export const MemberRolePopupContent = ({ roles }: MemberRolePopupContentProps) => {
-  const groupAddress = `#/working-groups/${groupNameToURLParam(roles[0].groupName)}`
+  const groupAddress = generatePath(`/working-groups/${groupNameToURLParam(roles[0].groupName)}`)
```

Comment:
I think that we can inline `/working-groups/${groupNameToURLParam(roles[0].groupName)}` because `generatePath()` should be used when you'd like to properly use params object:

```js
 generatePath("/user/:id/:entity(posts|comments)", {
  id: 1,
  entity: "posts"
});
```

With one parameter I think that with one parameter it is enough to use a template string.

Reply:
Yep, forgot to undo